tests/integration/test_integration.py

```diff
@@ -190,7 +190,6 @@ async def _deploy_prometheus_k8s(ops_test: OpsTest):
         await ops_test.model.deploy(
             PROMETHEUS_APP_NAME,
             application_name=PROMETHEUS_APP_NAME,
-            channel="edge",
             trust=True,
```

Comment:
Isn't the CI complaining about missing series in the `deploy` arguments? For instance `series="focal"`

Reply:
Nope.